Kudu.Core/SourceControl/Git/LibGit2SharpRepository.cs

```diff
@@ -239,9 +240,15 @@ public void FetchWithoutConflict(string remoteUrl, string branchName)
             }
             catch (LibGit2SharpException exception)
             {
-                if (exception.Message.Equals("Unsupported URL protocol"))
+                tracer.TraceWarning("LibGit2SharpRepository fetch failed with {0}", exception);
+
+                // LibGit2Sharp doesn't support SSH yet. Use GitExeRepository
+                // LibGit2Sharp only supports smart Http protocol
+                if (exception.Message.Equals("Unsupported URL protocol") ||
```

Comment:
if this is the only reason. we can write an specific log to MDS, and query MDS from time to time to collect the data.

because for people who use it, if there is another failure case, they will wait for our fix and deploy to unblocked them.